cmd/operator-sdk/generate/bundle/bundle.go

```diff
@@ -0,0 +1,217 @@
+// Copyright 2020 The Operator-SDK Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package bundle
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/operator-framework/operator-registry/pkg/lib/bundle"
+	"sigs.k8s.io/kubebuilder/pkg/model/config"
+
+	genutil "github.com/operator-framework/operator-sdk/cmd/operator-sdk/generate/internal"
+	gencsv "github.com/operator-framework/operator-sdk/internal/generate/clusterserviceversion"
+	"github.com/operator-framework/operator-sdk/internal/generate/collector"
+	"github.com/operator-framework/operator-sdk/internal/scaffold/kustomize"
+)
+
+// TODO(estroz): make these paths relative to inputDir
+const kubebuilderKustomization = `resources:
+- ../default
+- ../samples
+`
+
+func (c *bundleCmd) setDefaults(cfg *config.Config) {
+	if c.operatorName == "" {
+		c.operatorName = filepath.Base(cfg.Repo)
+	}
+}
+
+func (c bundleCmd) runKustomize(cfg *config.Config) error {
+
+	if !c.quiet {
+		fmt.Println("Generating bundle manifest kustomize bases")
+	}
+
+	defaultDir := filepath.Join("config", "bundle")
+	if c.inputDir == "" {
+		c.inputDir = defaultDir
+	}
+	if c.outputDir == "" {
+		c.outputDir = defaultDir
+	}
+	if c.apisDir == "" {
+		if cfg.MultiGroup {
+			c.apisDir = "apis"
+		} else {
+			c.apisDir = "api"
+		}
+	}
+
+	csvGen := gencsv.Generator{
+		OperatorName: c.operatorName,
+		OperatorType: genutil.PluginKeyToOperatorType(cfg.Layout),
+	}
+	opts := []gencsv.Option{
+		gencsv.WithBase(c.inputDir, c.apisDir),
+		gencsv.WithBaseWriter(c.outputDir),
+	}
+	if err := csvGen.Generate(cfg, opts...); err != nil {
+		return fmt.Errorf("error generating ClusterServiceVersion: %v", err)
+	}
+
+	if err := kustomize.WriteIfNotExist(c.outputDir, kubebuilderKustomization); err != nil {
+		return err
+	}
+
+	if !c.quiet {
+		fmt.Println("Bases generated successfully")
```

Comment:
```suggestion
		fmt.Println("Bundle manifest kustomize bases generated  successfully in: %v", c.outputDir)
```
It is important to know where the files have been added. 
Also, the nomenclature adopted here is not the same over above.